ruby/rb-aescrypt/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           ruby 1.0
+
+ruby.branches       3.3 3.2 3.1
+ruby.setup          {aescrypt aes-rb} 0.1.0 extconf.rb {README \
+                    aes-example.rb aes-test.rb aes-ecb-tbl.txt}
+revision            1
+maintainers         nomaintainer
+description         An AES encryption algorithm for Ruby, using the AESCrypt library
+long_description	An all-C (for performance) implementation of the AES \
```

Comment:
There may be an actual tab here that needs to be expanded, as per the modeline.

Reply:
@herbygillot Thanks, I will fix it.